src/zcl_abapgit_repo.clas.abap

```diff
@@ -56,6 +56,14 @@ CLASS zcl_abapgit_repo DEFINITION
         !io_dot_abapgit TYPE REF TO zcl_abapgit_dot_abapgit
       RAISING
         zcx_abapgit_exception .
+    METHODS get_dot_apack
+      RETURNING
+        VALUE(ro_dot_apack) TYPE REF TO zcl_abapgit_apack_reader .
+    METHODS set_dot_apack
```

Comment:
this method can be moved to private?

Reply:
I can move `set_dot_apack`, but `get_dot_apack` is used from `zcl_abapgit_services_repo`

Reply:
yes, only set_dot_apack should be set to private

Reply:
done in https://github.com/larshp/abapGit/pull/3285/commits/85566af6d3a0b8d286dca995ab6a2a9db987ac56